cg/store/models.py

```diff
@@ -987,3 +988,59 @@ class Order(Base):
 
     def to_dict(self):
         return to_dict(model_instance=self)
+
+
+class DigitisationUnit(Base):
```

Comment:
Describes the actual consumable. Fields such as consumable model (10B, S4, etc), expiration date etc.